### DISCUSSION TITLE
prisma.join is not a function

### DISCUSSION THREAD

**Discussion by sridivyatalluri** [prisma/prisma#10925] [Q&A] [answered]
Hi,

i am using   "@prisma/client": "^3.5.0", and i need following query using prisma raw SQL .

```
sub_users=[ 461, 565, 566, 707, 273 ]
select * from customers where customer_id in (sub_users);
```

```
const { PrismaClient } = require("@prisma/client");
const prisma = new PrismaClient();
prisma.$queryRaw`SELECT * FROM Customers o WHERE  o.customer_id IN (${prisma.join(sub_users)}) `;
```

When i execute the statement , i am getting "prisma.join is not a function";

Please suggest, where i am going wrong.

**Comment by sabinadams** [comment, marked as answer]
@sridivyatalluri The `.join` utility is not on the generated client, but on the Prisma namespace.

Something like this should work for you:

```ts
const { PrismaClient, Prisma } = require("@prisma/client");
const prisma = new PrismaClient();
prisma.$queryRaw`SELECT * FROM Customers o WHERE  o.customer_id IN (${Prisma.join(sub_users)}) `;
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
